ddtrace/llmobs/_writer.py

```diff
@@ -139,7 +139,7 @@ def enqueue(self, event: LLMObsSpanEvent) -> None:
         self._enqueue(event)
 
     def _data(self, events: List[LLMObsSpanEvent]) -> Dict[str, Any]:
-        return {"ml_obs": {"stage": "raw", "type": "span", "spans": events}}
+        return {"_dd.stage": "raw", "event_type": "span", "spans": events}
```

Comment:
we'll need a change for eval metrics too, right? ie `"event_type": "eval-metric"` in `LLMObsEvalMetricWriter._data`? or will this be addressed in a separate PR?

Reply:
We won't need to change eval metrics, since evaluation metrics are submitted to the Rapid GO API (this payload structure is only regarding our EVP endpoint). But yes, we'll need to change the eval metric payloads once we change eval metrics over to use the public EVP endpoint